src/pc/1.17/ChunkColumn.js

```diff
@@ -0,0 +1,275 @@
+const SmartBuffer = require('smart-buffer').SmartBuffer
+const ChunkSection = require('./ChunkSection')
+const constants = require('../common/constants')
+const varInt = require('../common/varInt')
+const BitArray = require('../common/BitArrayNoSpan')
+
+// wrap with func to provide version specific Block
+module.exports = (Block, mcData) => {
+  return class ChunkColumn {
+    constructor (options) {
+      this.minWorldHeight = options.minWorldHeight ?? 0
```

Comment:
If these properties are added to the api, they should be added to other versions as well with a constant value.